mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/dsl/syntax/DslSyntaxUtils.java

```diff
@@ -40,6 +43,10 @@
 
   private DslSyntaxUtils() {}
```

Comment:
Hard to extract this now, is used by several modules both in api and mule artifacts. 
Follow up ticket: MULE-12002